pkg/secretless/config/config.go

```diff
@@ -4,179 +4,25 @@ import (
 	"fmt"
 	"io/ioutil"
 	"regexp"
-	"strconv"
-
-	"github.com/go-ozzo/ozzo-validation"
 
+	crdAPIv1 "github.com/cyberark/secretless-broker/pkg/apis/secretless.io/v1"
```

Comment:
Bit on the fence but don't think this needed a change from `crd_api_v1` -> `crdAPIv1`